oxy/yaml/mpu.yaml

```diff
@@ -0,0 +1,51 @@
+!Messages
+msgs:
+- !CANMsg
+    id: "0x500"
+    desc: "NERduino Acceleromter"
```

Comment:
Don't call this NERduino anymore, we aren't using that board

Reply:
What should i call it

Reply:
Prob MPU Accelerometer